modules/API/API.js

```diff
@@ -1325,6 +1325,19 @@ class API {
         });
     }
 
+    /**
+     * Notify external application (if API is enabled) that an error was thrown.
+     *
+     * @param {Object} error - The error.
+     * @returns {void}
+     */
+    notifyErrorThrown(error: Object) {
+        this._sendEvent({
+            name: 'error-thrown',
```

Comment:
can we just call this "error" ? nothing is really thrown...

Reply:
done